### PR TITLE
Judging page

### LINKED ISSUE #3: [Feature] Create the judging page
### Description

Create the judging pages. There are two states for this page -- when the judge is viewing their first project (i.e. no previous project) and after the judge has seen a different project (perform pair-wise comparison)

### Designs

image
image

### PR DESCRIPTION
**Pull request by pm3512**
# Description

Adds judging page at /judging

Closes #3

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by gramliu** [comment]
image

Can we make the description text slightly smaller?

**Comment by gramliu** [comment]
image

Great job with the colors here. One problem I'm seeing though is that even though you have instructions, it's not immediately intuitive for users (who might not be super tech savvy) to do this. Could you try making the selection use radio buttons instead?

File: src/components/VotingList.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+// list multiple prizes with voting options
+export default function VotingList({
+  prizes,
+  project,
+  compareProjects,
+}: Props) {
+  // Default project selection color
+  const inactiveColor = "#000000";
```
Comment (gramliu) [inline src/components/VotingList.tsx:29]: Can we move these to `tailwind.config.cjs`?

See https://tailwindcss.com/docs/theme

File: src/components/VotingList.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                <label className="flex flex-row gap-2">
+                  <input
+                    type="radio"
+                    value={Votes.This}
+                    name={`prize${i}`}
+                    onChange={() => {
+                      updateVotes(i, Votes.This);
+                    }}
+                    checked={votes[i] === Votes.This}
+                    className="text-blue-600 focus:ring-blue-500 dark:focus:ring-blue-600 border-gray-300 bg-gray-100 focus:ring-2 dark:border-gray-600 dark:bg-gray-700 dark:ring-offset-gray-800"
```
Comment (gramliu) [inline src/components/VotingList.tsx:125]: Is the box around the radio button intended behavior?
image

**Review by gramliu — CHANGES_REQUESTED** [review]

File: src/components/VotingList.tsx (hunk 1 above)
Comment (gramliu) [reply src/components/VotingList.tsx:29]: By moving it to the tailwind config, we can assign the colors using tailwind classes

File: src/components/VotingList.tsx (hunk 2 above)
Comment (pm3512) [reply src/components/VotingList.tsx:125]: What browser are you using? It looks like this for me on google and firefox
image

Comment (gramliu) [reply src/components/VotingList.tsx:125]: I'm using Microsoft edge which should have similar behavior to chrome. It's fine then if it works for chrome and firefox

Comment (gramliu) [reply src/components/VotingList.tsx:125]: Anyway, most people will be viewing this on mobile

**Comment by gramliu** [comment]
[LGTM](https://lgtm.lol/i/9)

**Review by gramliu — APPROVED** [review]

**Pull request merged**